docs/sources/enterprise/access-control/assign-rbac-roles.md

```diff
@@ -73,7 +73,7 @@ Instead of using the Grafana role picker, you can use file-based provisioning to
 
 **Before you begin:**
 
-- [Enable role provisioning]({{< relref "./enable-rbac-and-provisioning#enable-role-provisioning" >}})
+- See [Role provisioning]({{< relref "./rbac-provisioning#rbac-provisioning" >}})
```

Comment:
For what purpose are we sending folks to these docs?

Reply:
Since the instructions for assignment requires knowledge about provisioning, I thought to keep this in place.